discord/scheduled_events.py

```diff
@@ -209,7 +209,7 @@ def __init__(
             ScheduledEventStatus, data.get("status")
         )
         self.subscriber_count: int | None = data.get("user_count", None)
-        self.creator_id = data.get("creator_id", None)
+        self.creator_id = int(data.get("creator_id", None))
```

Comment:
The API documentation states that this field is nullable:

> `creator_id` will be null and `creator` will not be included for events created before October 25th, 2021, when the concept of creator_id was introduced and tracked.

```suggestion
        self.creator_id: int | None = utils._get_as_snowflake(data, "creator_id")
```

Not tested.